Scripts/WikiUpdater.cs

```diff
@@ -62,93 +132,225 @@ public async Task<bool> Run(CancellationToken cancellationToken)
+            var untranslatedSections = sections.Select(s => UntranslateSection(s, untranslatedPageName)).ToList();
 
-            var untranslatedPage = new Wiki.Page($"WIKI_PAGE_{untranslatedOrganelleName}",
+            var untranslatedPage = new Wiki.Page($"WIKI_PAGE_{untranslatedPageName}",
                 internalName,
-                url,
-                untranslatedSections);
+                pageUrl,
+                untranslatedSections,
+                untranslatedInfobox,
+                noticeSceneName);
             var translatedPage = new Wiki.Page(name,
                 internalName,
-                url,
-                sections);
+                pageUrl,
+                sections,
+                translatedInfobox,
+                noticeSceneName);
 
-            organellePages.Add(new TranslationPair(untranslatedPage, translatedPage));
+            allPages.Add(new TranslationPair(untranslatedPage, translatedPage));
 
-            ColourConsole.WriteSuccessLine($"Populated content for organelle with internal name {internalName}");
+            ColourConsole.WriteSuccessLine($"Populated content for page with internal name {internalName}");
         }
 
-        return organellePages;
+        return allPages;
+    }
+
+    /// <summary>
+    ///   Extracts all the fields of an associated page's infobox and generates translation keys
+    /// </summary>
+    /// <param name="body">The body of the page</param>
+    /// <param name="internalName">Internal name of the page</param>
+    /// <returns>A tuple containing the translated and untranlsated versions of the detected fields</returns>
+    /// <exception cref="InvalidOperationException">Thrown when an infobox is not found</exception>
+    private (List<InfoboxField> Untranslated, List<InfoboxField> Translated)
+        GetInfoBoxFields(IHtmlElement body, string internalName)
+    {
+        ColourConsole.WriteInfoLine($"Extracting infobox content for page with internal name {internalName}");
+
+        // Get the infobox element
+        var infobox = body.QuerySelector(INFO_BOX_SELECTOR) ?? throw new InvalidOperationException(
+            $"Did not find infobox on page with internal name {internalName}");
+
+        var translated = new List<InfoboxField>();
+        var untranslated = new List<InfoboxField>();
+
+        foreach (var row in infobox.Children)
+        {
+            if (row.Children.Length <= 1)
+                continue;
+
+            // Parse the HTML table to get keys and values
+            var value = row.Children.Where(e => e.LocalName == "td").First();
+            var translatedKey = row.Children.Where(e => e.LocalName == "th").First().TextContent.Trim();
+            var id = value.Id;
+
+            if (id == null)
+                continue;
+
+            var textContent = value.TextContent.Trim();
+
+            if (textContent == internalName)
```

Comment:
Why is this skipped?

Reply:
`internalName` is its own variable in `Page` and shouldn't be translated either.